src/Microsoft.Graph.Cli.Core/utils/Constants.cs

```diff
@@ -16,6 +16,10 @@ public class Constants
         
         public const string DefaultTenant = "common";
 
+        public const string DefaultAuthority = "https://login.microsoftonline.com";
+
+        public const string DefaultCertificateScope = "https://graph.microsoft.com/.default";
```

Comment:
While working on moving the default authentication values, I've  built this from the request URI so we have support for national clouds out of the box (sheme://host/.default)

Reply:
The CLI doesn't have national cloud support at the moment.

Reply:
understood. The suggestion here is to build the code in such a way that it wouldn't need a rewrite when we want to add national cloud support.

Reply:
I've realized this was only necessary for private clients and I don't use it at the moment. I'll remove it for now.